x-pack/plugin/sql/qa/src/main/java/org/elasticsearch/xpack/sql/qa/jdbc/JdbcTestUtils.java

```diff
@@ -86,32 +87,30 @@ public static void logResultSetData(ResultSet rs, Logger log) throws SQLExceptio
 
         while (rs.next()) {
             sb.setLength(0);
-            for (int i = 1; i <= columns; i++) {
-                column.setLength(0);
-                if (i > 1) {
-                    sb.append(" | ");
-                }
-                sb.append(trimOrPad(column.append(rs.getString(i))));
-            }
+            rowAsString(rs, sb, column, columns);
             log.info(sb);
         }
     }
 
-    public static String resultSetCurrentData(ResultSet rs) throws SQLException {
+    static String resultSetCurrentData(ResultSet rs) throws SQLException {
         ResultSetMetaData metaData = rs.getMetaData();
         StringBuilder column = new StringBuilder();
 
         int columns = metaData.getColumnCount();
 
         StringBuilder sb = new StringBuilder();
+        rowAsString(rs, sb, column, columns);
+        return sb.toString();
+    }
+
+    private static void rowAsString(ResultSet rs, StringBuilder sb, StringBuilder column, int columns) throws SQLException {
```

Comment:
The method signature seems incorrect `column` seems to be used only locally and `sb` can be returned by the method - passing them as arguments doesn't offer any value.

Reply:
Will fix, just used the automatic refactoring.